laborator/content/operatii-memorie-gdb/7-pointers/pointers.c

```diff
@@ -24,9 +27,51 @@ int main() {
 	cu succes. In caz contrar, programul va crapa.
 	*/
 	// assert(SIGN(my_strcmp(s1, s2)) == SIGN(strcmp(s1, s2)));
-	// assert(my_memcpy(dest, src, sizeof(src)) == memcpy(dest, src, sizeof(src)));
-	// assert(my_strcpy(dest, src) == strcpy(dest, src));
-	free(dest);
+	// assert(test_strcpy(src, sizeof(src)));
+	// assert(test_memcpy(src, sizeof(src)));
 
 	return 0;
 }
+
+
+/* USED FOR TESTING! DO NOT MODIFY! */
+
+int test_strcpy(const char *src, size_t size) {
+	char *dest1 = malloc(size);
```

Comment:
Do some error handling in case the allocation fails (`malloc()` returns `NULL`).